ext/broadcast_channel/lib.rs

```diff
@@ -112,7 +112,7 @@ pub fn init<BC: BroadcastChannel + 'static>(
 ) -> Extension {
   Extension::builder(env!("CARGO_PKG_NAME"))
     .dependencies(vec!["deno_webidl", "deno_web"])
-    .js(include_js_files!(
+    .esm(include_js_files!(
       prefix "internal:ext/broadcast_channel",
```

Comment:
Discussed offline, relying on this `internal:` prefix here is brittle. We should be handling this automatically. I'm also wondering if we should remove `prefix` option completely and use extension name specified in `Extension::builder()`

Reply:
Agreed